02-generalize/Square.cpp

```diff
@@ -0,0 +1,67 @@
+/*Program illustrates the drawing of a Square Box  in C++ programming*/
+
+#include <iostream>
+using namespace std;
+
+class Square
+{
+
+private:
+  int column,row,side;
+  char symbol;
+
+public:
+  void square_generate(char function_symbol,int function_side)
+  {
+    symbol = function_symbol;
+    side = function_side;
+
+    if(cin)
+    {
+      try
+      {
+        if(side < 1)
+        throw side;
+        else if(side == 1)
+        top_bottomSide(side);
```

Comment:
**NEVER** mix case styles. `top_bottomSide` is a mix of both snake case & camel case. Use either: `top_bottom_side` or `topBottomSide`. Also, remember to avoid mixing case within the program.